checker.pl

```diff
@@ -80,8 +88,106 @@
 % TODO
 check_arts_req(Transcript, ModifiedTranscript).
 
-% TODO
-check_breadth_req(Transcript, ModifiedTranscript).
+% check_breadth_req(["MATH 100","CHEM 101","PHYS 101","BIOL 100","STAT 100","CPSC 100","ASTR 100"], MT).
+check_breadth_req(TR, MT):-
+    check_breadth_general(TR, MT1, "MATH", N1),
+    check_breadth_general(MT1, MT2, "CHEM", N2),
+    check_breadth_general(MT2, MT3, "PHYS", N3),
+    check_breadth_LFSC(MT3, MT4, N4),
+    check_breadth_STAT(MT4, MT5, N5),
+    check_breadth_general(MT5, MT6, "CPSC", N6),
+    check_breadth_EPSC(MT6, MT7, N7),
+    Total1 is N1+N2+N3+N4+N5+N6,
+    Total2 is N1+N2+N3+N4+N5+N6+N7,
+    Total2 > 5,
+    (   Total1 > 5
+    ->  MT = MT6
+    ;   Total2 > 5
+    ->  MT = MT7
+    ;   MT = TR 
+    ),
+    writeln("Breadth Requirement is Satisfied.").
```

Comment:
We should've named this `check_remaining_breadth_req()`

I think you are performing the most general breadth requirement check, but many of these requirements will be satisfied by the core courses check. Take a look at the degree navigator version > 2018, and use the **Remaining Science Breadth** section as a reference.

It's gonna be something like:
chem, phys, lfs ; chem lfs eosc ; physic lfs eosc ; physc eosc  chem
```
(check_breadth(chem, MT, 3), <- number of chem courses sum to 3 credits atleast
check_breadth(phys, MT, 3),
check_breadth(lfs, MT, 3));

(check_breadth(chem, MT, 3),
check_breadth(eosc, MT, 3),
check_breadth(lfs, MT, 3));

(check_breadth(phys, MT, 3),
check_breadth(eosc, MT, 3),
check_breadth(lfs, MT, 3)).
```

Reply:
Right, I've changed the breadth check to just chemistry, physics, life science, and earth&physical science groups. I also added a check for the credits being 3 or more.